src/styles/components/main.scss

```diff
@@ -0,0 +1,5 @@
+*{
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
Don't use * selector, font-family specify into html or body, reset margins only for elements that really need it.